dasklearn/functions.py

```diff
@@ -58,7 +59,10 @@ def aggregate(settings: SessionSettings, params: Dict):
         model_manager.process_incoming_trained_model(idx, model)
 
     start_time = time.time()
-    agg_model = model_manager.aggregate_trained_models()
+    if weights:
```

Comment:
Is this if-else construction necessary? Even if `weights` is `None`, it should work right?